styles/web/_vars.scss

```diff
@@ -750,1222 +250,975 @@ $sprk-btn-tertiary-border: $sprk-btn-border-width $sprk-btn-border-style
+  $sprk-link-border-bottom-style $sprk-link-has-icon-underline-color !default;
+/// The color value for Links with icons in the hover state.
+$sprk-link-has-icon-hover-color-text: $sprk-purple-deep !default;
+/// The fill color for icons contained in Links on hover.
+$sprk-link-has-icon-hover-fill: $sprk-link-has-icon-hover-color-text !default;
+/// The stroke color for icons contained in Links on hover.
+$sprk-link-has-icon-hover-stroke: $sprk-link-has-icon-hover-color-text !default;
+/// The border settings for Links with icons in the hover state.
+$sprk-link-has-icon-hover-border-bottom: $sprk-link-has-icon-underline-width
+  $sprk-link-border-bottom-style $sprk-link-has-icon-underline-color !default;
+/// The disabled Link
+$sprk-link-disabled-color: $sprk-black-tint-50 !default;
+/// The Disabled Link border-bottom
+$sprk-link-disabled-border-bottom-color: $sprk-link-disabled-color !default;
+/// The underline width for the disabled Link.
+$sprk-link-disabled-underline-width: 0 !default;
+/// The underline settings for the disabled Link..
+$sprk-link-disabled-border-bottom: $sprk-link-disabled-underline-width
+  $sprk-link-border-bottom-style $sprk-link-disabled-color !default;
+/// The disabled Link color on hover.
+$sprk-link-disabled-hover-color: $sprk-black-tint-50 !default;
+/// The disabled Link hover border-bottom
+$sprk-link-disabled-hover-border-bottom-color: $sprk-link-disabled-color !default;
+/// The underline settings for the disabled Link..
+$sprk-link-disabled-hover-border-bottom: $sprk-link-disabled-underline-width
+  $sprk-link-border-bottom-style $sprk-link-disabled-color !default;
+/// The default margin left indentation on Lists.
+$sprk-list-margin-left: $sprk-space-m !default;
+/// The default margin left indentation on Lists.
+$sprk-list-indented-margin-left: $sprk-space-l !default;
 /// The width of the menu icon on narrow viewports in the Masthead.
 $sprk-masthead-menu-icon-width: $sprk-icon-xl !default;
 /// The height of the menu icon on narrow viewports in the Masthead.
 $sprk-masthead-menu-icon-height: $sprk-icon-xl !default;
 /// The background color of the Masthead.
 $sprk-masthead-bg-color: $sprk-white !default;
-/// The padding of the Masthead content, menu, and branding sections.
+/// The padding around the content section in the Masthead on wide viewports.
 $sprk-masthead-content-padding: $sprk-space-s !default;
-/// The padding top around the content items in the
-/// content section of the Masthead.
+/// The padding of the Masthead content, menu, and branding sections.
+$sprk-masthead-content-padding-wide: $sprk-space-m !default;
+/// The padding top around the content items in the content section of the Masthead.
 $sprk-masthead-content-item-padding-top: $sprk-space-s !default;
-/// The padding bottom around the content
-/// items in the content section of the Masthead.
+/// The padding top around the content items in the content section of the Masthead in wide viewports..
```

Comment:
Lets fix up this comment, add a period etc in the masthead.json file

Reply:
I got this!

Reply:
DONE! 

https://github.com/sparkdesignsystem/spark-design-system/pull/3663/files#diff-b386748607d222684a98521f211e7dfcdb3343818fe299d737de352730544050